src/main/java/com/palantir/gradle/revapi/RevapiPlugin.java

```diff
@@ -59,7 +59,7 @@ public void apply(Project project) {
                 .register("revapiAnalyze", RevapiAnalyzeTask.class, task -> {
                     Configuration revapiNewApi = project.getConfigurations().create("revapiNewApi", conf -> {
                         conf.extendsFrom(
-                                project.getConfigurations().getByName(JavaPlugin.RUNTIME_ELEMENTS_CONFIGURATION_NAME));
```

Comment:
generally extending *Elements configurations is bad in any case, as they are meant to be consumed by other projects.